makiflow/layers/trainable_layers.py

```diff
@@ -260,15 +261,26 @@ def __init__(self, D, name):
             Dimension of bias vector.
         name : str
             Name of this layer.
+        trainable : bool
+            If true, bias will be learned in train, otherwise its will be constant variable.
+        b : numpy array
+            Bias' weights. This value is used for the bias initialization with pretrained bias.
         """
         self.D = D
         self.name = name
+        self.trainable = trainable
+
+        if b is None:
+            b = np.zeros(D)
+        elif b.shape[0] != D:
+            raise ValueError(f"Initial value of `b` should have same dimension size like D={D}")
```

Comment:
`The` initial value of `b` should `must have the same` dimension size `as` D={D}"